SpywareFilter/sections/specific.txt

```diff
@@ -52,6 +52,7 @@
 !
 /analytics.js$domain=freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|newsmax.com|bolgehastanesi.com|ewtc.de|newsdig.tbs.co.jp
 !
+||dbfukofby5ycr.cloudfront.net^
```

Comment:
Please move it to the `tracking_servers_firstparty.txt`.
If you have doubts about adding rules to the files, look at the beginning of the file - there are examples.